llama_stack/providers/adapters/inference/ollama/ollama.py

```diff
@@ -38,6 +40,7 @@ def __init__(self, url: str) -> None:
         self.url = url
         tokenizer = Tokenizer.get_instance()
         self.formatter = ChatFormat(tokenizer)
+        self.model = "Llama3.2-1B-Instruct"
```

Comment:
not intended?

Reply:
It's intended. Do you think it's good to have a default model?

Reply:
no we should not have a default model. this is provided by the rest of the infrastructure at runtime. the inference providers don't have any notions of default models.

Reply:
Just made another commit. Lmk if any modification is needed!